docs/language/learn-ql/index.rst

```diff
@@ -1,21 +1,21 @@
-Learning QL
-###########
+Learning TEST
+###############
 
 
-`QL <https://semmle.com/ql>`__ is the query language used in Semmle's `variant analysis <https://semmle.com/variant-analysis>`__ engine. 
-You can use queries written in QL to explore code and quickly find variants of security vulnerabilities and bugs. 
-The QL language is also part of the technology behind `LGTM <https://lgtm.com>`__,  Semmle's analysis platform that combines deep semantic code search with data science insights to help developers ship secure code.
+`TEST <https://semmle.com/ql>`__ is the query language used by Semmle's `variant analysis <https://semmle.com/variant-analysis>`__ engine. 
```

Comment:
I suspect that this sentence needs to be rewritten, but in the short term changed the first word because we're explicitly applying the term to code.

Reply:
I think that it would be good to spend a day or two reworking the homepage (index), introduction-to-ql, and about-ql topics. I don't think it would take too long, and would be easier than attempting to adapt the current content to distinguish between the language and the CodeQL product.